.github/workflows/build-pdf.yml

```
@@ -39,7 +39,7 @@ jobs:
             wget -O notoserif.zip "https://noto-website-2.storage.googleapis.com/pkgs/NotoSerif-hinted.zip"
             wget -O notoserifcjk.zip "https://noto-website-2.storage.googleapis.com/pkgs/NotoSerifCJKsc-hinted.zip"
             wget -O notosanscjk.zip "https://noto-website-2.storage.googleapis.com/pkgs/NotoSansCJKsc-hinted.zip"
-            wget "https://mirrors.ctan.org/fonts/cm-unicode.zip"
+            wget "https://mirrors.ctan.org/fonts/cm-unicode.zip" --no-check-certificate
```

```suggestion
            wget "https://tug.ctan.org/fonts/cm-unicode.zip"
```

要不换这个？看起来也是挂在 `ctan.org` 下的，（希望）应该不会有证书问题了

---

还是换之前的普林斯顿吧，那个已经稳定跑了很久了，感觉是一个“已知可用”的东西。

---

```suggestion
            wget "https://mirror.math.princeton.edu/pub/CTAN/fonts/cm-unicode.zip" --no-check-certificate
```